linz.js

```diff
@@ -196,7 +196,7 @@ Linz.prototype.configure = function() {
 
         debugGeneral('Connecting to the database');
 
-        _this.mongoose.connect(_this.get('mongo'));
+        _this.mongoose.connect(_this.get('mongo'), _this.get('mongoOptions'));
```

Comment:
I think we need some sort of `Object.assign({}, _this.get('mongoOptions'), the options from the passed in mongoose)` here. Does Linz already do something like that @smebberson ?

Reply:
@allanchau, so the `_this.get('mongo')` and `_this.get('mongoOptions')` just pulls values from the defaults, unless they've been overrided at init time. What benefit would `Object.assign` have here?

Reply:
@allanchau, also, just note, this won't get called if the passed in Mongoose is already connected.

Reply:
Ah ok, this is all good then.

Reply:
🎉